mantras.py

```diff
@@ -0,0 +1,46 @@
+import discord
+import logging
+from discord.utils import get
+from channels import REPETITIONS
+from roles import HIVE_MXTRESS
+
+class Mantras():
+
+    current_mantra = ""
+
+    async def load_mantra(self):
+        if Mantras.current_mantra != "": return True #No need to load a mantra if it's already there.
+        with open("data/current_mantra.txt", "r") as mantra_file:
+            Mantras.current_mantra = mantra_file.readline()
+            self.LOGGER.info(f"Mantra loaded from file: {Mantras.current_mantra}")
+            mantra_file.close()
+
+    async def update_mantra(self, message):
+
+        print("CURRENT MANTRA IS:")
+        print(Mantras.current_mantra)
+
+        if message.content.startswith("Repeat :: ") == False: return False
+        new_mantra = message.content.replace("Repeat :: ","")
+        with open("data/current_mantra.txt", "w") as mantra_file:
+            mantra_file.write(new_mantra)
+            Mantras.current_mantra = new_mantra #Update the global mantra.
+            topic_message = message.content.replace("Repeat :: ", "Repeat :: [ID] :: ")
+            await message.channel.edit(topic=topic_message) #Update the channel description
+            self.LOGGER.info(f"Mantra updated and written to file: {Mantras.current_mantra}")
+            mantra_file.close()
+
+        print("UPDATED MANTRA IS:")
+        print(Mantras.current_mantra)
```

Comment:
Beware! `print()` leads to problems when the server is running for some time. Please always use the logger.